services/QuillLMS/app/services/skills_report/question_responses_for_classroom.rb

```diff
@@ -21,10 +21,12 @@ class QuestionResponsesForClassroom < ApplicationService
             sc.student_id AS user_id,
             cr.activity_session_id,
             cr.question_number,
-            cr.extra_metadata->>'question_uid' as question_uid,
+            COALESCE(cr.extra_metadata->>'question_uid',#{' '}
```

Comment:
Out of scope for this PR, but does this skills report include Evidence Activities? Do we need to do something for those (they don't have question_uids).

Reply:
Good call-out.  This code update will include Evidence questions, but only if their associated concepts are flagged for the report, which I think they are not intended to be currently.  It will also treat those questions as "proofreader" questions, but since `question_type` is only used to determine if we need to do special Fill in the Blank scoring, it shouldn't matter if we ever do decide to include those Concepts in the report.